acid/features/history/service.py

```diff
@@ -34,13 +34,14 @@ def pagination(number_of_buildsets, page, per_page, page_links):
 
 
 class BuildSetsPaginated:
-    def __init__(self, pipeline, per_page):
+    def __init__(self, pipeline, max_branch_filter_size, per_page):
         self.pipeline = pipeline
         self.per_page = per_page
+        self.max_branch_filter_size = max_branch_filter_size
         self.query = self._create_query()
 
     def __len__(self):
-        return len(self.query)
+        return self.query.count()
```

Comment:
This fails in CI. I'm not totally sure why, but it looks like Pony catch it as other variant of `count()`. `len` should be ok or check it.